scripts/package/postinstall.sh

```diff
@@ -119,6 +119,13 @@ finish_permissions() {
     # We also change the owner of the binary to observiq-otel-collector
     chown -R observiq-otel-collector:observiq-otel-collector /opt/observiq-otel-collector/observiq-otel-collector /opt/observiq-otel-collector/plugins/*
     chmod 0640 /opt/observiq-otel-collector/plugins/*
+
+  # Initialize the log file to ensure it is owned by observiq-otel-collector.
+  # This prevents the service (running as root) from assigning ownership to
+  # the root user. By doing so, we allow the user to switch to observiq-otel-collector
+  # user for 'non root' installs.
+  touch /opt/observiq-otel-collector/log/collector.log
+  chown observiq-otel-collector:observiq-otel-collector /opt/observiq-otel-collector/log/collector.log
```

Comment:
Is there any concern about log file rotation here? Like, when it rotates, does collector.log get new permissions or retain the old file's permissions? Or is the same file reused? I'm not sure how the rotation works ATM.